app/src/components/market/sections/market_create/steps/items/create_scalar_market.tsx

```diff
@@ -125,6 +120,52 @@ export const CreateScalarMarket = (props: Props) => {
     upperBound,
   } = props
 
+  let lowerBoundNumber: string
+  let startingPointNumber: string
+  let upperBoundNumber: string
+
+  if (startingPoint !== null) {
+    startingPointNumber = formatBigNumber(startingPoint, 18, 2)
+  }
+
+  if (lowerBound !== null) {
+    lowerBoundNumber = formatBigNumber(lowerBound, 18, 2)
+  }
+
+  if (upperBound !== null) {
+    upperBoundNumber = formatBigNumber(upperBound, 18, 2)
+  }
+
+  const [lowerBoundError, setLowerBoundError] = useState('')
+  const [startingPointError, setStartingPointError] = useState('')
+  const [upperBoundError, setUpperBoundError] = useState('')
+
+  const handleLowerBoundError = (value: BigNumberInputReturn) => {
+    Number(value.formattedValue) <= 0
+      ? setLowerBoundError('Value must be greater than 0')
+      : Number(value.formattedValue) > Number(upperBoundNumber) && Number(upperBoundNumber) !== 0
+      ? setLowerBoundError(`Value must be less than ${upperBoundNumber}`)
+      : setLowerBoundError('')
+  }
+
+  const handleStartingPointError = (value: BigNumberInputReturn) => {
+    Number(value.formattedValue) <= 0
+      ? setStartingPointError('Value must be greater than 0')
```

Comment:
For `startingPoint` and `upperBound`, we can remove the > 0 check because they have to be greater than `lowerBound` anyway